### PR TITLE
fix: supertokens google oauth 

### LINKED ISSUE #23: Authentication Doesn't Works
The authentication is done with SuperTokens. Currently there should be GIthub Social Login. In Future, other recipes can be implemented

### PR DESCRIPTION
**Pull request by DarkPhoenix2704**
Closes #23

### CODE REVIEW

**Review by sreehari2003 — COMMENTED** [review]
Frontend looks good , @reenphygeorge  need a review for backend

File: .env
Comment (reenphygeorge) [inline .env:1]: Make it fossfolio instead of postgres

File: apps/web/.env
Comment (reenphygeorge) [inline apps/web/.env:1]: this .env too ?

File: apps/api/src/main.ts
Comment (reenphygeorge) [inline apps/api/src/main.ts:1]: Also it would be better to change the port from 8080 if we are integrating NocoDB as 8080 is the default port of Noco...

**Review by reenphygeorge — CHANGES_REQUESTED** [review]

File: apps/api/src/main.ts
Comment (DarkPhoenix2704) [reply apps/api/src/main.ts:1]: We won't be using noco

File: .env
[the remark this answers is not in the document]
Comment (DarkPhoenix2704) [reply .env:1]: ok

**Pull request closed without merging**